src/Sets/Zonotope.jl

```diff
@@ -554,29 +554,36 @@ end
 # Zonotope vertex enumeration methods
 # ====================================
 
-function _vertices_list_2D(c::AbstractVector{N}, G::AbstractMatrix{N}; apply_convex_hull::Bool) where {N}
-    if same_sign(G)
-        return _vertices_list_2D_positive(c, G)
-    else
-        # FIXME generalized 2D vertices list function is not implemented yet
-        # See LazySets#2209
-        return _vertices_list_iterative(c, G, apply_convex_hull=apply_convex_hull)
-    end
+@inline function _angles(point::AbstractVector{N})  where{N}
+    (atan(point[2], point[1]) / π * 180 + 360) % 360
 end
 
-function _vertices_list_2D_positive(c::AbstractVector{N}, G::AbstractMatrix{N}) where {N}
-    n, p = size(G)
-
-    # TODO special case p = 1 or p = 2 ?
+function _single_quadrant_vertices_enum(G::AbstractMatrix{N}, sorted::Bool=true) where{N}
+    if !sorted
+        G = sortslices(G, dims=2, by=_angles)
+    end
+    return VPolygon(2*cumsum(hcat(G, -G), dims=2) .- sum(G, dims=2))
+end
 
-    sorted_G = sortslices(G, dims=2, by=x->atan(x[2], x[1]))
-    index = ones(N, p, 2*p)
-    @inbounds for i in 1:p
-        index[i, i+1:i+p-1] .= -one(N)
+function _vertices_list_2D(c::AbstractVector{N}, G::AbstractMatrix{N}; apply_convex_hull::Bool) where {N}
+    if apply_convex_hull
+        return _vertices_list_iterative(c, G, apply_convex_hull)
```

Comment:
```suggestion
        return _vertices_list_iterative(c, G, apply_convex_hull=apply_convex_hull)
```
see https://travis-ci.org/github/JuliaReach/LazySets.jl/jobs/730769538#L678